README.md

```diff
@@ -37,7 +37,6 @@ This project contains the following top level components:
 * [OpenTelemetry API](api/):
   * [stable apis](api/all/src/main/java/io/opentelemetry/api/) including `Tracer`, `Span`, `SpanContext`, and `Baggage`
   * [semantic conventions](semconv/) Generated code for the OpenTelemetry semantic conventions.
-  * [context api](api/context/src/main/java/io/opentelemetry/context/) The OpenTelmetry Context implementation.
```

Comment:
context api directory does not exist

Reply:
we shouldn't delete this line. Let's correct it to the correct directory:
`context/src/main/java/io/lopentelemetry/context/`

Reply:
@wsargent bump!

Reply:
looking

Reply:
done